src/nvim/api/buffer.c

```diff
@@ -14,6 +14,7 @@
 #include "nvim/api/private/defs.h"
 #include "nvim/vim.h"
 #include "nvim/buffer.h"
+#include "nvim/change.h"
```

Comment:
Should I look into removing / remove `#include "nvim/misc1.h"` where `change.h` is included now?

Reply:
I wouldn't bother until we finish removing `misc1.c` completely (would be a nice milestone--we removed `misc2.c` years ago, but `misc1.c` has been clinging to life for too long :)